sw/device/tests/spi_host_flash_test_impl.c

```diff
@@ -119,17 +119,28 @@ status_t test_page_program(dif_spi_host_t *spi) {
   return OK_STATUS();
 }
 
-status_t test_page_program_quad(dif_spi_host_t *spi, uint8_t opcode) {
-  enum {
+status_t test_page_program_quad(dif_spi_host_t *spi, uint8_t opcode,
+                                uint8_t address_width) {
+  enum { kPageSize = 256, kAddress = kPageSize * 10 };
+  TRY_CHECK(address_width == 1 || address_width == 2 || address_width == 4);
+  dif_spi_host_width_t addr_width = kDifSpiHostWidthStandard;
+  switch (address_width) {
+    case 4:
+      addr_width = kDifSpiHostWidthQuad;
+      break;
+    case 2:
+      addr_width = kDifSpiHostWidthDual;
+    case 1:
```

Comment:
Missing `break`?

Reply:
It's intentional, because the case 1 is the value initialized, but I just realized that it would be much clearer if I remove the `case 1`.